docs/getting-started/function-next-steps.md

```diff
@@ -0,0 +1,58 @@
+# Next Steps with Knative Functions
```

Comment:
I don't see this linked into the nav anywhere -- intended?

Reply:
Currently it's only pointed to in the Next Steps document. I think ultimately this content (probably modified) should be in the main Functions section of the site once that has been created.